io-engine/src/logger.rs

```diff
@@ -420,26 +425,45 @@ impl LogFormat {
 ///
 /// We might want to suppress certain messages, as some of them are redundant,
 /// in particular, the NOTICE messages as such, they are mapped to debug.
-pub fn init_ex(level: &str, format: LogFormat) {
+pub fn init_ex(level: &str, format: LogFormat, events_url: Option<url::Url>) {
     // Set up a "logger" that simply translates any "log" messages it receives
     // to trace events. This is for our custom spdk log messages, but also
     // for any other third party crates still using the logging facade.
+
     LogTracer::init().expect("failed to initialise LogTracer");
 
     // Create a default subscriber.
-    let builder = tracing_subscriber::fmt::Subscriber::builder()
+    let builder = tracing_subscriber::fmt::layer()
         .with_span_events(FmtSpan::FULL)
         .event_format(format);
 
-    let subscriber = match EnvFilter::try_from_default_env() {
-        Ok(filter) => builder.with_env_filter(filter).finish(),
-        Err(_) => builder.with_env_filter(level).finish(),
+    let filter = match EnvFilter::try_from_default_env() {
```

Comment:
Should we filter out "mbus-events-target" from stdout?

Reply:
Implemented excluding eventing traces now.